.github/workflows/koupleless_runtime_release.yml

```
@@ -132,23 +132,42 @@ jobs:
         run: |
           # 创建一个映射来存储模块名到 JDK 版本的映射
           modules=(${{ join(fromJSON(steps.find-maven-modules.outputs.modules-list), ' ') }})
-          modules_in_right_jdk=""
+          modules_in_right_jdk_array=()
           for module in "${modules[@]}"; do
             # 读取模块中的 pom.xml 来确定 JDK 版本
             jdk_version=$(grep -m 1 '<java.version>' $module/pom.xml | sed 's/<[^>]*>//g' | xargs)
             echo "${module} JDK version: ${jdk_version}"
-            # 如果是目标 jdk 版本，则执行 release 操作
-          
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)